packages/data/src/create-selector.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import createSel from 'rememo';
+
+const NULL_OBJ = {};
+const TRUE_OBJ = {};
+const FALSE_OBJ = {};
+
+export function createSelector< S extends ( ...args: any[] ) => any >(
+	sel: S,
+	dep: any
```

Comment:
This seems to slightly change the signature interface. The [original](https://github.com/aduth/rememo/blob/master/rememo.js#L124) allows the second arg to be omitted (`undefined`), while this would require a falsey value to be passed explicitly if we want to fallback to the `arrayOf` fallback [here](https://github.com/aduth/rememo/blob/master/rememo.js#L129). Just double-checking :)